utils/grafana-data-source/src/lib.rs

```diff
@@ -31,7 +31,7 @@ use parking_lot::RwLock;
 mod types;
```

Comment:
I am a bit confused. In some places we replace the Grafana logic, in some places it still exists. We should be consistent, me advocating for replacing Grafana. What do you all think?

Reply:
Agreed.

Reply:
me too Agreed.